gallery/verifier/Cargo.toml

```
@@ -7,6 +7,8 @@ license-file = "../../LICENSE"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+concordium-rust-sdk.workspace = true
+concordium_base = "8.0.0"
```

If we keep the submodule dependency then it is easier to update if we don't have hard coded versions.

```suggestion
concordium-rust-sdk.workspace = true
concordium_base = "*"
```